app/src/main/java/org/fossasia/openevent/dbutils/DbContract.java

```diff
@@ -68,12 +66,33 @@ public DbContract() {
                         + ORGANISATION + TEXT_TYPE + COMMA_SEP
                         + POSITION + TEXT_TYPE + COMMA_SEP
                         + COUNTRY + TEXT_TYPE + COMMA_SEP
-                        + SESSIONS + TEXT_TYPE + COMMA_SEP
                         + " )";
         public static final String DELETE_TABLE = "DROP TABLE IF EXISTS " + TABLE_NAME;
 
     }
 
+    public static abstract class Sessions_speakers implements BaseColumns{
```

Comment:
I did format the code. What notation did I not follow, please tell me so I can improve.